client/components/line-chart/docs/example.js

```diff
@@ -1,11 +1,9 @@
-/** @format */
-
 /**
  * External dependencies
  */
 import React, { Component } from 'react';
-import { moment } from 'i18n-calypso';
 import { range, random } from 'lodash';
+import moment from 'moment';
```

Comment:
No localized moment is needed here, since we're just calculating some dates and grabbing their Unix timestamps.